From 7fdf80a9e8fcf0a769fda3641abbf6966f89b31a Mon Sep 17 00:00:00 2001
Message-Id: <7fdf80a9e8fcf0a769fda3641abbf6966f89b31a.1635086064.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Sun, 24 Oct 2021 16:32:56 +0200
Subject: [PATCH] Build with tesseract/leptonica on Fedora

Others might want to do the same as soon as mupdf is packaged with OCR
support.

Signed-off-by: Michael J Gruber <mjg@fedoraproject.org>
---
 setup.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/setup.py b/setup.py
index 1f0520d..b7f3c18 100644
--- a/setup.py
+++ b/setup.py
@@ -43,6 +43,8 @@ OPENSUSE = ARCH_LINUX + [
 ]
 FEDORA = ARCH_LINUX + [
     "harfbuzz",
+    "leptonica",
+    "tesseract",
 ]
 NIX = ARCH_LINUX + ["harfbuzz"]
 LIBRARIES = {
-- 
2.33.1.1061.g4ab8308fcf

